recipes/lagraph/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "8Aug2020" %}
```

Comment:
@conda-forge/staged-recipes @conda-forge/core, do we have thoughts on the version here?

Reply:
In order for the user to get the latest version, it should be `2020.08.08` or similar

Reply:
Thanks.  Changed to `2020.08.08`.